test/e2e/tests/confirmations/signatures/permit.spec.ts

```diff
@@ -103,11 +107,40 @@ describe('Confirmation Signature - Permit @no-mmi', function (this: Suite) {
           primaryType: 'Permit',
           uiCustomizations: ['redesigned_confirmation', 'permit'],
           location: 'confirmation',
+          decodingChangeTypes: ['RECEIVE', 'LISTING'],
+          decodingResponse: 'CHANGE',
         });
       },
       mockSignatureRejected,
     );
   });
+
+  it('display decoding information if available', async function () {
+    await withTransactionEnvelopeTypeFixtures(
+      this.test?.fullTitle(),
+      TransactionEnvelopeType.legacy,
+      async ({ driver }: TestSuiteArguments) => {
+        await unlockWallet(driver);
+        await openDapp(driver);
+        await driver.clickElement('#signPermit');
```

Comment:
Can we use the existing helpers to abstract this as we do in the other tests, such as `openDappAndTriggerSignature`?

Ideally this would be using a page object model but out of scope for this PR.

Reply:
done